transactions/create_treasury.cdc

```diff
@@ -8,8 +9,10 @@ transaction(initialSigners: [Address], initialThreshold: UInt64) {
 
     // Seed Treasury with commonly used vaults
     let flowVault <- FlowToken.createEmptyVault()
+    let fusdVault <- FUSD.createEmptyVault()
```

Comment:
Can we update FE as well to create all supported Vault FLOW/FUSD/USDC ? https://github.com/DapperCollectives/VESSEL/blob/main/packages/client/src/flow/initializeTreasury.tx.js#L11

Reply:
Can it be done as part of https://linear.app/dappercollectives/issue/VSL-52/add-fusd-usdc-flow?

Reply:
sure, that sounds good! 👍